src/pages/responsible-trading/_trading-limits.js

```diff
@@ -214,9 +215,13 @@ const TradingLimits = () => {
                                     </TimelineTick.Item>
                                     <TimelineTick.Item>
                                         <Text>
-                                            {localize(
-                                                'If you do not wish to renew the self-exclusion and you make a request to trade again, there will be a cooling-off period of 1 day before you are allowed access to our site. Please note that email is insufficient and your request must be made by phone.',
-                                            )}
+                                            {isUK
```

Comment:
I think these changes should be pushed to `crowdin` branch first for translation purpose

suggestion:
1. submit PR to `crowdin` branch ( see the tutorial on the readme file)
2. after making the changes, dont forget to run `npm run translate:extract' 
3. commit changes and submit PR to crowdin

Reply:
Alright, noted. Will be incorporated. Thank you.